graph/src/util/log.rs

```diff
@@ -103,3 +104,37 @@ pub fn register_panic_hook(panic_logger: Logger, shutdown_sender: oneshot::Sende
         process::exit(1);
     }));
 }
+
+pub enum LogCode {
+    SubgraphStartFailure,
+    SubgraphSyncingFailure,
+    SubgraphSyncingFailureNotRecorded,
+    BlockIngestionStatus,
+    GraphQlQuerySuccess,
+    GraphQlQueryFailure,
+}
+
+impl Display for LogCode {
+    fn fmt(&self, f: &mut Formatter) -> Result<(), Error> {
+        let value = match self {
+            LogCode::SubgraphStartFailure => "SubgraphStartFailure",
+            LogCode::SubgraphSyncingFailure => "SubgraphSyncingFailure",
+            LogCode::SubgraphSyncingFailureNotRecorded => "SubgraphSyncingFailureNotRecorded",
+            LogCode::BlockIngestionStatus => "BlockIngestionStatus",
+            LogCode::GraphQlQuerySuccess => "GraphQLQueryTimer",
```

Comment:
These two are not used yet; is this planned?

Reply:
They are now being used !